### PR TITLE
Fixes #10223: Add rudder version properties in parent pom

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/10223

### CODE REVIEW

**Comment by peckpeck** [comment]
Commit modified

File: pom.xml [hunk 1]
```diff
@@ -281,6 +281,8 @@ limitations under the License.
 
     <!-- Libraries version that MUST be used in all children project -->
 
+    <current-year>2017</current-year>
+    <rudder-major-version>4.1</rudder-major-version>
     <rudder-version>4.1.0~beta4-SNAPSHOT</rudder-version>
```
Comment (fanf) [inline pom.xml]: does that works:
`      <rudder-version>${rudder-major-version}.0~beta4-SNAPSHOT</rudder-version> `

And if yes, do we want it to be like that?

Comment (fanf) [inline pom.xml]: will likely breaks the release script, thought

Comment (peckpeck) [inline pom.xml]: Yes, and it probably won't work because this is the probable reason of the existence of the next line.

Comment (peckpeck) [inline pom.xml]: and it's not really readable

**Comment by peckpeck** [comment]
OK, merging this PR

**Pull request merged**